### PR TITLE
Submission of SDMtune

### PR DESCRIPTION
**Pull request by sgvignali**
## env_collinearity
* `corVar()`: prints the name of correlated variables and the relative correlation coefficient value
* `plotCor()`: plots a correlation matrix heat map
* `varSel()`: performs a data-driven variable selection to remove highly correlated variables.

## bias
* `thinData()`: removes all but one location per raster cell

## data_partitioning
* `randomFolds()`: creates random folds for cross validation

## mod_fit
* `train()`: trains a model using one of the following methods: Artificial Neural Networks, Boosted Regression Trees, Maxent, Maxnet or Random Forest
* `gridSearch()`: given a set of possible hyperparameter values, the function trains models with all the possible combinations of hyperparameters
* `randomSearch()`: performs a random search in the hyperparameters space, creating a population of random models each one with a random combination of the provided hyperparameters values
* `optimizeModel()`: uses a Genetic Algorithm implementation to optimize the model hyperparameter configuration according to the chosen metric

## mod_evaluate
* `aicc()`: computes the Akaike Information Criterion corrected for small samples size
* `auc()`: computes the AUC using the Man-Whitney U Test formula
* `tss`(): computes the max TSS of a given model
* `confMatrix()`: computes Confusion Matrixes for threshold values varying from 0 to 1

## pred_general
* `predict()`: predicts the output for a new dataset given a trained model

## pred_extrapolation
* `predict()`: predicts the output for a new dataset given a trained model
* `plotPred()`: plots predictions
* `plotPA()`: plots a presence absence map using the given threshold

## pred_inspect
* `plotResponse()`: plots the Response Curve of the given environmental variable
* `plotVarImp()`: plots the variable importance as a bar plot
* `plotROC()`: plots the ROC curve of the given model and print the AUC value
* `plotJK()`: plots the Jackknife Test for variable importance
* `doJK()`: runs the Jackknife test for variable importance removing one variable at time
* `varImp()`: randomly permutes one variable at time and computes the decrease in training AUC
* `modelReport()`: creates a report that shows the main results

### CODE REVIEW

**Comment by sylvainschmitt** [comment]
Thanks for your submission. @adamlilith or @jamiemkass, is one of you okay to review it?

**Review by sylvainschmitt — APPROVED** [review]
Everything looks great. Thank your for the clean submission and the clear details about the available functions. If the automatic check approve the submission it's good to go for me.

**Comment by sylvainschmitt** [comment]
Congratulations, the automatic checks are all good. So as far as I'm concerned, you're good to go. I'm just going to give Adam and Jamie time to try to get two reviews. But if not, it's all good and I'll add your PR. I will probably ask you as a reviewer for future submissions.

**Comment by sgvignali** [comment]
Ok, thanks for developing the app!

**Comment by sylvainschmitt** [comment]
Ok we will have a lot of review to do coming and I think one is enough regarding all the detail you gave so I'll merge your PR.
